Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -9223,6 +9226,82 @@ def inner(client, args):
 
     return dec
 
+def generate_random_64_string() -> str:
```

Comment:
Don't use python3 syntax.
We haven't officially dropped support for python2 scripts/integrations.
Add docstring or add it to the object ignore list in `Documentation/common_server_docs.py`.
```suggestion
def generate_random_64_string():
```

Reply:
Not relevant